### PR TITLE
Added pricing in text so AI can search and redirect to pricing websit…

### PR DESCRIPTION
**Pull request by eshanrnh**
…e from docs

## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 10/umbraco-commerce/installation/licensing-model.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ This is an add-on domain for existing licenses. Refunds will not be given for th
 
 ## Configuring your license
 
-You can purchase a license via the [Umbraco Commerce](https://umbraco.com/products/umbraco-commerce/) project page. A member of the [SUITS team](mailto:suits@umbraco.com) will manage this process. In the process, you will need to provide all domains you wish to have covered by the license such as primary and staging/qa domains. You should then receive a license code to be installed in your solution.
+You can look at the pricing, features, and purchase a license at [Umbraco Commerce](https://umbraco.com/products/add-ons/commerce/) page. A member of the [SUITS team](mailto:suits@umbraco.com) will manage this process. In the process, you will need to provide all domains you wish to have covered by the license such as primary and staging/QA domains. You should then receive a license code to be installed in your solution.
```
Comment (alina-tincas) [inline 10/umbraco-commerce/installation/licensing-model.md:48]:
```suggestion
You can look at the pricing, features, and purchase a license on the [Umbraco Commerce](https://umbraco.com/products/add-ons/commerce/) page. A member of the [sales team](mailto:suits@umbraco.com) will manage this process. In the process, you will need to provide all domains you wish to have covered by the license such as primary and development/staging/QA domains. You should then receive a license code to be installed in your solution.
```

File: 10/umbraco-deploy/installation/the-licensing-model.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ The license for Umbraco Deploy comes with a recurring yearly fee. Learn more abo
 
 ## Configuring your license
 
-You can purchase the license at [Umbraco Deploy Licenses](https://umbraco.com/products/umbraco-deploy/)
+You can look at the pricing, plans, features, and purchase the license at [Umbraco Deploy On-premises](https://umbraco.com/products/add-ons/deploy/umbraco-deploy-on-premises/) page.
```
Comment (alina-tincas) [inline 10/umbraco-deploy/installation/the-licensing-model.md:49]:
```suggestion
You can look at the pricing, plans, features, and purchase the license on the [Umbraco Deploy On-premises](https://umbraco.com/products/add-ons/deploy/umbraco-deploy-on-premises/) page.
```

File: 10/umbraco-forms/installation/the-licensing-model.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ This is an add-on domain for existing licenses. Refunds will not be given for th
 
 ## Configuring your license
 
-You can purchase the license at [Umbraco Forms Licenses](https://umbraco.com/products/umbraco-forms/).
+You can look at the pricing, features, and purchase the license at [Umbraco Forms](https://umbraco.com/products/add-ons/forms/) page.
```
Comment (alina-tincas) [inline 10/umbraco-forms/installation/the-licensing-model.md:52]:
```suggestion
You can look at the pricing, features, and purchase the license on the [Umbraco Forms](https://umbraco.com/products/add-ons/forms/) page.
```

File: 10/umbraco-workflow/installation/licensing.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ Umbraco Workflow is a licensed product that does not require a purchase. New ins
 
 ## Purchasing an Umbraco Workflow License
 
-If you want to buy an Umbraco Workflow license, reach out to the sales team at [**suits@umbraco.com**](mailto:suits@umbraco.com). Existing Plumber license holders who wish to upgrade to Umbraco Workflow should contact [**suits@umbraco.com**](mailto:suits@umbraco.com).
+You can look at the pricing, plans, and features at [Umbraco Workflow](https://umbraco.com/products/add-ons/workflow/) page. If you want to buy an Umbraco Workflow license, reach out to the sales team at [**suits@umbraco.com**](mailto:suits@umbraco.com). Existing Plumber license holders who wish to upgrade to Umbraco Workflow should contact [**suits@umbraco.com**](mailto:suits@umbraco.com).
```
Comment (alina-tincas) [inline 10/umbraco-workflow/installation/licensing.md:7]:
```suggestion
You can look at the pricing, plans, and features on the [Umbraco Workflow](https://umbraco.com/products/add-ons/workflow/) page. If you want to buy an Umbraco Workflow license, reach out to the sales team at [**suits@umbraco.com**](mailto:suits@umbraco.com). Existing Plumber license holders who wish to upgrade to Umbraco Workflow should contact [**suits@umbraco.com**](mailto:suits@umbraco.com).
```

File: 12/umbraco-commerce/installation/licensing-model.md [hunk 5]
```diff
@@ -45,7 +45,7 @@ This is an add-on domain for existing licenses. Refunds will not be given for th
 
 ## Configuring your license
 
-You can purchase a license via the [Umbraco Commerce](https://umbraco.com/products/umbraco-commerce/) project page. A member of the [SUITS team](mailto:suits@umbraco.com) will manage this process. In the process, you will need to provide all domains you wish to have covered by the license such as primary and staging/qa domains. You should then receive a license code to be installed in your solution.
+You can look at the pricing, features, and purchase a license at [Umbraco Commerce](https://umbraco.com/products/add-ons/commerce/) page. A member of the [SUITS team](mailto:suits@umbraco.com) will manage this process. In the process, you will need to provide all domains you wish to have covered by the license such as primary and staging/QA domains. You should then receive a license code to be installed in your solution.
```
Comment (alina-tincas) [inline 12/umbraco-commerce/installation/licensing-model.md:48]:
```suggestion
You can look at the pricing, features, and purchase a license on the [Umbraco Commerce](https://umbraco.com/products/add-ons/commerce/) page. A member of the [sales team](mailto:suits@umbraco.com) will manage this process. In the process, you will need to provide all domains you wish to have covered by the license such as primary and development/staging/QA domains. You should then receive a license code to be installed in your solution.
```

File: 12/umbraco-deploy/installation/the-licensing-model.md [hunk 6]
```diff
@@ -44,7 +44,7 @@ The license for Umbraco Deploy comes with a recurring yearly fee. Learn more abo
 
 ## Purchasing your license
 
-You can purchase the license at [Umbraco Deploy Licenses](https://umbraco.com/products/umbraco-deploy/)
+You can look at the pricing, plans, features, and purchase the license at [Umbraco Deploy On-premises](https://umbraco.com/products/add-ons/deploy/umbraco-deploy-on-premises/) page.
```
Comment (alina-tincas) [inline 12/umbraco-deploy/installation/the-licensing-model.md:47]:
```suggestion
You can look at the pricing, plans, features, and purchase the license on the [Umbraco Deploy On-premises](https://umbraco.com/products/add-ons/deploy/umbraco-deploy-on-premises/) page.
```

File: 12/umbraco-forms/installation/the-licensing-model.md [hunk 7]
```diff
@@ -47,7 +47,7 @@ This is an add-on domain for existing licenses. Refunds will not be given for th
 
 ## Configuring your license
 
-You can purchase the license at [Umbraco Forms Licenses](https://umbraco.com/products/umbraco-forms/).
+You can look at the pricing, features, and purchase the license at [Umbraco Forms](https://umbraco.com/products/add-ons/forms/) page.
```
Comment (alina-tincas) [inline 12/umbraco-forms/installation/the-licensing-model.md:50]:
```suggestion
You can look at the pricing, features, and purchase the license on the [Umbraco Forms](https://umbraco.com/products/add-ons/forms/) page.
```

File: 12/umbraco-workflow/installation/licensing.md [hunk 8]
```diff
@@ -4,7 +4,7 @@ Umbraco Workflow is a licensed product that does not require a purchase. New ins
 
 ## Purchasing an Umbraco Workflow License
 
-If you want to buy an Umbraco Workflow license, reach out to the sales team at [**suits@umbraco.com**](mailto:suits@umbraco.com). Existing Plumber license holders who wish to upgrade to Umbraco Workflow should contact [**suits@umbraco.com**](mailto:suits@umbraco.com).
+You can look at the pricing, plans, and features at [Umbraco Workflow](https://umbraco.com/products/add-ons/workflow/) page. If you want to buy an Umbraco Workflow license, reach out to the sales team at [**suits@umbraco.com**](mailto:suits@umbraco.com). Existing Plumber license holders who wish to upgrade to Umbraco Workflow should contact [**suits@umbraco.com**](mailto:suits@umbraco.com).
```
Comment (alina-tincas) [inline 12/umbraco-workflow/installation/licensing.md:7]:
```suggestion
You can look at the pricing, plans, and features on the [Umbraco Workflow](https://umbraco.com/products/add-ons/workflow/) page. If you want to buy an Umbraco Workflow license, reach out to the sales team at [**suits@umbraco.com**](mailto:suits@umbraco.com). Existing Plumber license holders who wish to upgrade to Umbraco Workflow should contact [**suits@umbraco.com**](mailto:suits@umbraco.com).
```

File: 13/umbraco-commerce/installation/the-licensing-model.md [hunk 9]
```diff
@@ -45,7 +45,7 @@ This is an add-on domain for existing licenses. Refunds will not be given for th
 
 ## Configuring your license
 
-You can purchase a license via the [Umbraco Commerce](https://umbraco.com/products/umbraco-commerce/) project page. A member of the [SUITS team](mailto:suits@umbraco.com) will manage this process. In the process, you will need to provide all domains you wish to have covered by the license such as primary and staging/qa domains. You should then receive a license code to be installed in your solution.
+You can look at the pricing, features, and purchase a license at  [Umbraco Commerce](https://umbraco.com/products/add-ons/commerce/) page. A member of the [SUITS team](mailto:suits@umbraco.com) will manage this process. In the process, you will need to provide all domains you wish to have covered by the license such as primary and staging/QA domains. You should then receive a license code to be installed in your solution.
```
Comment (alina-tincas) [inline 13/umbraco-commerce/installation/the-licensing-model.md:48]:
```suggestion
You can look at the pricing, features, and purchase a license on the [Umbraco Commerce](https://umbraco.com/products/add-ons/commerce/) page. A member of the [sales team](mailto:suits@umbraco.com) will manage this process. In the process, you will need to provide all domains you wish to have covered by the license such as primary and development/staging/QA domains. You should then receive a license code to be installed in your solution.
```

File: 13/umbraco-deploy/installation/the-licensing-model.md [hunk 10]
```diff
@@ -44,7 +44,7 @@ The license for Umbraco Deploy comes with a recurring yearly fee. Learn more abo
 
 ## Purchasing your license
 
-You can purchase the license at [Umbraco Deploy Licenses](https://umbraco.com/products/umbraco-deploy/)
+You can look at the pricing, plans, features, and purchase the license at [Umbraco Deploy On-premises](https://umbraco.com/products/add-ons/deploy/umbraco-deploy-on-premises/) page.
```
Comment (alina-tincas) [inline 13/umbraco-deploy/installation/the-licensing-model.md:47]:
```suggestion
You can look at the pricing, plans, features, and purchase the license on the [Umbraco Deploy On-premises](https://umbraco.com/products/add-ons/deploy/umbraco-deploy-on-premises/) page.
```

File: 13/umbraco-forms/installation/the-licensing-model.md [hunk 11]
```diff
@@ -47,7 +47,7 @@ This is an add-on domain for existing licenses. Refunds will not be given for th
 
 ## Configuring your license
 
-You can purchase the license at [Umbraco Forms Licenses](https://umbraco.com/products/umbraco-forms/).
+You can look at the pricing, features, and purchase the license at [Umbraco Forms](https://umbraco.com/products/add-ons/forms/) page.
```
Comment (alina-tincas) [inline 13/umbraco-forms/installation/the-licensing-model.md:50]:
```suggestion
You can look at the pricing, features, and purchase the license on the [Umbraco Forms](https://umbraco.com/products/add-ons/forms/) page.
```

File: 13/umbraco-workflow/installation/licensing.md [hunk 12]
```diff
@@ -4,7 +4,7 @@ Umbraco Workflow is a licensed product that does not require a purchase. New ins
 
 ## Purchasing an Umbraco Workflow License
 
-If you want to buy an Umbraco Workflow license, reach out to the sales team at [**suits@umbraco.com**](mailto:suits@umbraco.com). Existing Plumber license holders who wish to upgrade to Umbraco Workflow should contact [**suits@umbraco.com**](mailto:suits@umbraco.com).
+You can look at the pricing, plans, and features at [Umbraco Workflow](https://umbraco.com/products/add-ons/workflow/) page. If you want to buy an Umbraco Workflow license, reach out to the sales team at [**suits@umbraco.com**](mailto:suits@umbraco.com). Existing Plumber license holders who wish to upgrade to Umbraco Workflow should contact [**suits@umbraco.com**](mailto:suits@umbraco.com).
```
Comment (alina-tincas) [inline 13/umbraco-workflow/installation/licensing.md:7]:
```suggestion
You can look at the pricing, plans, and features on the [Umbraco Workflow](https://umbraco.com/products/add-ons/workflow/) page. If you want to buy an Umbraco Workflow license, reach out to the sales team at [**suits@umbraco.com**](mailto:suits@umbraco.com). Existing Plumber license holders who wish to upgrade to Umbraco Workflow should contact [**suits@umbraco.com**](mailto:suits@umbraco.com).
```

File: umbraco-cloud/getting-started/umbraco-cloud-plans.md [hunk 13]
```diff
@@ -41,3 +41,5 @@ Umbraco Cloud Professional plan
 - Disk - 10400 MB
 
 These are hard plan quotas, which cannot be exceeded for more than a 5-minute interval. When a cloud environment surpasses the CPU or memory limits defined by the plan quota, the app service hosting the environment will automatically restart. If the app service experiences multiple restarts, we classify it as a noisy neighbor. Then it will be relocated to a dedicated environment to guarantee that other tenants in the shared pool are not negatively impacted.
+
+You can look at the pricing, plans, and features at [Umbraco Cloud Pricing](https://umbraco.com/products/umbraco-cloud/pricing/) page.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/umbraco-cloud-plans.md:45]:
```suggestion
You can look at the pricing, plans, and features on the [Umbraco Cloud Pricing](https://umbraco.com/products/umbraco-cloud/pricing/) page.
```

File: umbraco-heartcore/README.md [hunk 14]
```diff
@@ -22,7 +22,7 @@ How to create a Heartcore Trial
 
 All Umbraco Heartcore projects include a Content Delivery Network (CDN) using CloudFlare. This CDN is used for caching content and media fetched through the Content Delivery API. Additionally, the media CDN (media.umbraco.io) allows for resizing and cropping options, which improves both performance and stability.
 
-You can read more about all the features and benefits on the [Umbraco Heartcore product page](https://umbraco.com/products/umbraco-heartcore/).
+You can read more about all the features and benefits on the [Umbraco Heartcore product page](https://umbraco.com/products/umbraco-heartcore/). You can also look at the pricing, plans, and features at [Umbraco Heartcore Pricing](https://umbraco.com/products/umbraco-heartcore/pricing/) page.
```
Comment (alina-tincas) [inline umbraco-heartcore/README.md:25]:
```suggestion
You can read more about all the features and benefits on the [Umbraco Heartcore product page](https://umbraco.com/products/umbraco-heartcore/). You can also look at the pricing, plans, and features on the [Umbraco Heartcore Pricing](https://umbraco.com/products/umbraco-heartcore/pricing/) page.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @eshanrnh thank you for the PR! I have some small suggestions, can you please look into them? 

Also, it seems that you have not added anything for UI-builder: https://umbraco.com/products/add-ons/ui-builder/ there is no pricing as of yet, but they still need to reach out to the form for a pricing as it mentions that licenses are sold per backoffice and to remove some restrictions you need to have a license: https://docs.umbraco.com/umbraco-ui-builder/installation/licensing-model

**Comment by eshanrnh** [comment]
Thanks @alina-tincas for your review 🙌 I've added pricing info for UI Builder as well 💪

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Looks good 💪

**Pull request merged**